server/src/core/pipeline/services/pipeline-stage.service.ts

```diff
@@ -35,4 +36,58 @@ export class PipelineStageService {
 
   // GroupBy
   groupBy = this.prismaService.pipelineStage.groupBy;
+
+  // Customs
+  async createDefaultPipelineStages({
+    workspaceId,
+    pipelineId,
+  }: {
+    workspaceId: string;
+    pipelineId: string;
+  }) {
+    return this.createMany({
```

Comment:
Although it looks like the right place to create the pipeline and stage instances, we may want to keep the seed data separated into separated json files.